packages/augur-sdk/src/state/getter/Users.ts

```diff
@@ -473,6 +472,41 @@ export class Users {
       }
     );
 
+    // Set unclaimedProceeds & unclaimedProfit
+    const marketsData = await db.findMarkets({
+      selector: { market: { $in: marketIds } },
+    });
+    for (const marketData of marketsData) {
+      marketTradingPositions[marketData.market].unclaimedProceeds = '0';
+      marketTradingPositions[marketData.market].unclaimedProfit = '0';
+      if (marketData.reportingState === MarketReportingState.Finalized || MarketReportingState.AwaitingFinalization) {
+        if (marketData.tentativeWinningPayoutNumerators) {
+          const outcome = marketData.tentativeWinningPayoutNumerators.findIndex((item: string) => Number(item) > 0);
```

Comment:
This won't work for scalars

Reply:
Yeah I'll use calculatePayoutNumeratorsValue instead

Reply:
No, I mean in scalars there are multiple non zero outcome values

Reply:
calculatePayoutNumeratorsValue won't return an outcome that can be used to look up the TokenBalanceChangedLog? How do you suggest getting the outcome to use for looking up the logs?

Reply:
You will have multiple outcomes, not a single outcome. For any non zero outcome in `tentativeWinningPayoutNumerators` you would retrieve the associated log for the market and that outcome. I describe this below

Reply:
Ok, I see now. Makes sense.